### ISSUE TITLE
How do I include Sourcekitten in Command Line tool

### ISSUE THREAD

**Issue by KarthikS941** [jpsim/SourceKitten#519]
I am trying to integrate Sourcekitten to a Command Line Tool in Xcode 9. I am getting, `Library not found error`.

Checked with a couple of links online, I came to know that there is a difference in how frameworks are included in app bundles and tool bundles. 

Is there a way in Source Kitten workspace, that we can generate a framework that's compatible with command line tools?

**Comment by jpsim** [comment]
You're right that there are a number of ways to include a Swift library in a command line tool. You'll want to link SourceKittenFramework to your binary, preferably statically for simpler deployments.

Take a look at any number of the projects included in the [README's "Projects Built With SourceKitten" section](https://github.com/jpsim/SourceKitten#projects-built-with-sourcekitten) to see how different projects have done this.

**Comment by KarthikS941** [comment]
thanks this helped. Started with Swift Package manager and it works like a charm.
